src/components/Tezori3.tsx

```diff
@@ -18,12 +18,13 @@ export function Tezori3() {
             <AddressBar />
                 <main>
                     <Routes>
-                        <Route path="/gallery"  element={<Gallery />}/>
-                        <Route path="/wallet"   element={<Wallet />}/>
-                        <Route path="/settings" element={<Settings />}/>
-                        <Route path="*"          element={<Navigate to="/gallery" replace />} /> 
+                            <Route path="/"         element={ <Navigate to="/gallery    " /> } />
```

Comment:
It's probably nothing, but why all the spaces after `"/gallery"`

Reply:
Thank you, resolved!